projects/core/components/svg/svg-options.ts

```diff
@@ -31,7 +31,7 @@ export interface TuiSvgOptions {
     readonly contentProcessor: TuiHandler<TuiSafeHtml, TuiSafeHtml>;
     readonly deprecated: TuiStringHandler<string>;
     readonly iconsPlace: string;
-    readonly path: TuiStringHandler<string>;
+    readonly path: (name: string, baseHref: string) => string;
```

Comment:
breaking change? should we support old variant?

Reply:
fixed